src/components/Dialog/src/DialogLayer.vue

```diff
@@ -175,6 +175,7 @@ export default {
 }
 
 .disableScroll {
+	position: fixed;
```

Comment:
This is a change outside of the pin-input component, but it prevents unnecessary body scrolling and jumpiness on dialogs when using the pin-input component inside of them.